### PR TITLE
some fixes and doc extensions

### PR DESCRIPTION
**Pull request by wistefan**
Hi Chris,
thanks a lot for you contribution. I did run it and work a little on it. I decided to do a PR to your repo, instead of thousand comments:)
- added some resources that where missing(mostly namespaces)
- extended the doc
- changed orion-ld & mongodb to be taken from the marinera platform 

The open question from my perspective is the integration of argocd with your keycloak. Whats the benefit of it? I did connect it to the openshift oauth directly on my local system, which feels easier.

### CODE REVIEW

File: kustomize/bundles/argocd/base/argocds/argocd/argocd.yaml [hunk 1]
```diff
@@ -4,6 +4,8 @@ metadata:
   name: argocd
   namespace: argocd
 spec:
```
Comment (wistefan) [inline kustomize/bundles/argocd/base/argocds/argocd/argocd.yaml:6]: Enable auth with the openshift accounts

File: kustomize/README.md [hunk 2]
```diff
@@ -112,3 +129,21 @@ path "/fiware/data/*" {
 - Visit https://argocd-server-argocd.apps-crc.testing/settings/clusters/https%3A%2F%2Fkubernetes.default.svc
 - Click [ Edit ]
```
Comment (wistefan) [inline kustomize/README.md:130]: Why is this required? For now, argo manages all namespaces.

**Review by computate — APPROVED** [review]
I love it @wistefan , thanks for your help to clean this up!

File: kustomize/bundles/argocd/base/argocds/argocd/argocd.yaml (hunk 1 above)
Comment (computate) [reply kustomize/bundles/argocd/base/argocds/argocd/argocd.yaml:6]: It's fine to leave out Oauth authentication here.

File: kustomize/README.md (hunk 2 above)
Comment (computate) [reply kustomize/README.md:130]: I am not very good at ArgoCD yet, so I'm sure I don't know what I'm doing :)
